mcs/class/System/Test/System.Diagnostics/SourceSwitchTest.cs

```diff
@@ -53,7 +53,7 @@ public class SourceSwitchTest
 		public void ConstructorNullName ()
 		{
 			SourceSwitch s = new SourceSwitch (null);
-			AssertHelper.IsEmpty (s.DisplayName);
+			Assert.AreEqual (s.DisplayName.Length, 0);
```

Comment:
or `Assert.That (s.DisplayName, Is.Empty)` <- but I'm not sure if mono's current nunit accept that syntax